src/App.jsx

```diff
@@ -1,18 +1,32 @@
 import reactLogo from "./assets/react.svg";
 import viteLogo from "/vite.svg";
 import "./App.css";
-import { useState } from "react"; 
-import TodoList from "./TodoList";
+import { useState, useEffect } from "react";
+import TodoList from "./TodoList.jsx";
```

Comment:
You shouldn't need this .jsx extension since you're using a bundler (In this case vite) 👍

Reply:
got it! thank you :)